### PR TITLE
demo: add alt text to examples logo

### PR DESCRIPTION
**Pull request by thbt**
Before submitting a pull request, please make sure you have at least performed the following:

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [ ] added/updated any applicable tests.
 - [ ] added/updated any applicable API documentation.
 - [ ] added/updated any applicable demos.

### CODE REVIEW

File: demo/src/app/components/shared/example-box/example-box.component.html [hunk 1]
```diff
@@ -6,10 +6,10 @@
       </span>
       <div class="float-right">
         <button class="btn-link" (click)="toggleCode()" title="{{showCode ? 'Hide' : 'Show'}} source code">
-          <img src="img/code-icon.png" height="24" width="24"/>
+          <img src="img/code-icon.png" height="24" width="24" alt="Source code icon"/>
         </button>
         <a target="_blank" href="app/components/{{component}}/demos/{{demo}}/plnkr.html" title="Edit in Plnkr" >
-          <img src="img/plunker-icon.png" height="24" width="24"/>
+          <img src="img/plunker-icon.png" height="24" width="24" alt="Plnkr icon"/>
```
Comment (wesleycho) [inline demo/src/app/components/shared/example-box/example-box.component.html]: I would call this `Plunker icon`

Comment (thbt) [inline demo/src/app/components/shared/example-box/example-box.component.html]: Should I rename the button title as well ?

Comment (wesleycho) [inline demo/src/app/components/shared/example-box/example-box.component.html]: Sure, makes sense.

**Comment by pkozlowski-opensource** [comment]
LGTM

**Pull request closed without merging**